apps/rentals_app.rb

```diff
@@ -0,0 +1,38 @@
+require_relative './handle_data'
+require_relative '../classes/rental'
+require_relative '../classes/person'
+require_relative '../classes/book'
+require_relative './books_app'
+require_relative './persons_app'
+require 'json'
+
+class RentalsApp
+  attr_accessor :rentals, :books, :persons
+
+  def initialize(books, persons)
+    @books = books
+    @persons = persons
+    @rentals = JSON.parse(File.read('./data/rentals.json'))
+  end
+
+  def add_rental
+    puts '********** Create a new rental **********'
+    puts 'Select which book you want to rent by entering its index: '
+    book_index = gets.chomp.to_i
+    puts 'Select a person from the list by its index: '
+    person_index = gets.chomp.to_i
```

Comment:
- [ ] **Optional:** While a user attempts to create a rental, consider displaying the available options to help them.